plugins/org.eclipse.n4js.flowgraphs/src/org/eclipse/n4js/flowgraphs/ControlFlowType.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2017 Marcus Mews.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Marcus Mews - Initial API and implementation
+ */
+package org.eclipse.n4js.flowgraphs;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import org.eclipse.n4js.flowgraphs.model.CatchToken;
+import org.eclipse.n4js.flowgraphs.model.JumpToken;
+import org.eclipse.n4js.n4JS.BreakStatement;
+import org.eclipse.n4js.n4JS.ContinueStatement;
+import org.eclipse.n4js.n4JS.ReturnStatement;
+import org.eclipse.n4js.n4JS.ThrowStatement;
+
+/**
+ * Defines the kind of the control flow. Control flow types are also used to specify {@link JumpToken}s and
+ * {@link CatchToken}s.
+ */
+public enum ControlFlowType {
+	/** Return edges are caused by {@link ReturnStatement}s */
+	Return,
+	/** Throw edges are caused by {@link ThrowStatement}s */
+	Throw,
+	/** Break edges are caused by {@link BreakStatement}s */
+	Break,
+	/** Continue edges are caused by {@link ContinueStatement}s */
+	Continue,
+	/** Successor edges are default edges */
```

Comment:
Since Successor is the default edge, move it to the beginning or end of the enum list for readability

Reply:
done